src/worker/main.ts

```diff
@@ -92,9 +96,8 @@ Object.keys(queues).forEach((name) => {
 		Sentry.setTag("queue", name);
 		Sentry.captureException(err);
 
-		const tags = [`name:${name}`];
-
-		statsd.increment("queue_error", tags);
+		const tags = [`queue:${name}`, `err:${err}`];
+		statsd.increment(metricError.queueError, tags);
```

Comment:
Err might contain parameters which potentially explode the cardinality. (like installationId or jobId)

We need to remove this tag, it ie enough to have this information in the log only